src/object/firefly.hpp

```diff
@@ -36,10 +36,14 @@ class Firefly final : public MovingSprite
   virtual std::string get_display_name() const override { return _("Checkpoint"); }
   virtual ObjectSettings get_settings() override;
 
+  virtual void on_flip(float height) override;
+
 private:
   SpritePtr m_sprite_light;
   bool activated;
   Vector initial_position; /**< position as in level file. This is where Tux will have to respawn, as the level is reset every time */
+
+private:
   void reactivate();
```

Comment:
(If we want to be strict with code quality, this function should go with the other private functions above, and the [non-copyable footer](https://github.com/SuperTux/supertux/blob/master/CODINGSTYLE.md#classes) is missing. But I won't put the responsibility on you, it's only if you feel motivated to do it :smile:)

Reply:
There aren't any other private functions in `firefly.hpp`, I just separated the _one_ from variables. But I will at least move it up to be under the public functions rather than the private variables.